project_structure.txt

```diff
@@ -1,31 +1,47 @@
-├── README.md          <- The top-level README for developers using this project.
-├── data
-│   ├── external       <- Data from third party sources.
-│   ├── interim        <- Intermediate data that has been transformed.
-│   ├── processed      <- The final, canonical data sets for modeling.
-│   └── raw            <- The original, immutable data dump.
-│
-├── models             <- Trained and serialized models, model predictions, or model summaries
-│
-├── notebooks          <- Jupyter notebooks.
-│
-├── references         <- Data dictionaries, manuals, and all other explanatory materials.
-│
-├── requirements.txt   <- The requirements file for reproducing the analysis environment, e.g.
-│                         generated with `pip freeze > requirements.txt`
-│
-├── setup.py           <- Make this project pip installable with `pip install -e`
-├── src                <- Source code for use in this project.
-│   ├── __init__.py    <- Makes src a Python module
-│   │
-│   ├── data           <- Scripts to download or generate data
-│   │   └── make_dataset.py
-│   │
-│   ├── features       <- Scripts to turn raw data into features for modeling
-│   │   └── build_features.py
-│   │
-│   ├── models         <- Scripts to train models and then use trained models to make
-│   │   │                 predictions
-│   │   ├── predict_model.py
-│   │   └── train_model.py
-└──
\ No newline at end of file
+├── README.md               <- The top-level README for developers using this project.
+├── requirements.txt        <- The requirements file for reproducing the analysis environment, e.g.  generated with `pip freeze > requirements.txt`                      
+├── run.py                  <- the file used to run the back-end serve.
+├── setup.py                <- Make this project pip installable with `pip install -e`
+└── Road  
+    ├── __init__.py         <- Makes Road a Python module
+    ├── data
+    │   ├── external        <- Data from third party sources.
+    │   ├── interim         <- Intermediate data that has been transformed.
+    │   ├── processed       <- The final, canonical data sets for modeling.
+    │   └── raw             <- The original, immutable data dump.
+    |
+    ├── endpoints
+    │   └── endpoints       <- script containing all endpoints in the server
+    |
+    ├── logs                <- folder conaining all logs files generated from the project
+    │   └── RouadStatus.log       
+    ├── models              <- Trained and serialized models, model predictions, or model summaries
```

Comment:
I recommend adding folders for pickled models and models notebooks inside the models folder

Reply:
these folders could be added when we go through the project if we need them , i dont think we need to add them from now and don't use them later  , but if the others agree with the idea there's no problem with adding them

Reply:
I agree with Zubaida, for more organizing...